content/blog/posts/visual-studio-2022-update.md

```diff
@@ -0,0 +1,277 @@
+Furthermore all O3DE build warnings and errors using the the v143 toolset were fixed and future VS2022 compile failures are protected against via the Automated Review pipeline validating each Pull Request that goes into O3DE repo [development](https://github.com/o3de/o3de) branch.  
+
+
+### Project Manager Updates
+The 22.10.0 release also included a feature update to the [Project Manager](https://www.o3de.org/docs/user-guide/project-config/project-manager/#projects) application. The requirement building against the Visual Studio 2019 application has been removed.  
+The Project Manager application will use the build toolset of latest version of Visual Studio installed on the user's machine for configuring and building O3DE Projects.  If the user only has Visual Studio 2019 installed with the v142 toolset, that will be used for compiling O3DE. If the user only has Visual Studio 2022 installed with the v143 toolset that will be used for compilation. Finally if the user has both Visual Studio 2019 and 2022 installed then Visual Studio 2022 v143 toolset will be used.
+![Project Manager Project Options](/images/blog/vs2022/project-manager-build-options.png)
+![Project Manager Building](/images/blog/vs2022/project-manager-building.png)
+
+
+## VS2022 vs VS2019 O3DE differences
+
+The are several differences to using Visual Studio 2022 to build O3DE versus Visual Studio 2019.  
+
+### VS2022(v143) uses more memory than VS2019(v142)
+
+The most notable is that Visual Studio 2022 uses more memory when compiling O3DE than Visual Studio 2019.  
+
+{{< note >}} If the Windows OS becomes unresponsive or build times start to expand to over several hours, this is indication that build is memory limited and users should check how much RAM they have available when building.
+A good way to mitigate build memory issues with the build using Visual Studio 2022 is to adjust the compilation parameters for compiling multiple projects in parallel(`/m:<N>` or `/maxcpucount:<N>`) and multiple cl.exe processes per project(`/p:CL_MPCount=<N>`).
+{{< /note >}}  
+
+When building from the command line on a machine with 64 logical cores and 64 GiB of ram, the parameters of `/maxcpucount` and `/p:CL_MPCount` can be supplied to CMake build wrapper command(`cmake --build`) to control the number of cl.exe processes that get launched.
+The following command restricts msbuild to build no more than 8 Visual Studio C++ Projects(.vcxproj) in parallel at a given time and no more than 16 `cl.exe` process per .vcxproj at once.
+```
+cmake --build build/windows --target Editor --config profile -- /maxcpucount:8 /p:CL_MPCount=16
+```
+
+The equivalent options in the Visual Studio 2022 IDE can be set via following IDE options
+* Setting the maximum number of parallel project builds value in the `Tools -> Options -> Projects and Solutions -> Build and Run` menu.  
+![Build and Run](/images/blog/vs2022/vs2022-parallel-project-builds-8.png)
```

Comment:
`![Build and Run](/images/blog/vs2022/vs2022-parallel-project-builds-8.png)`
Change to
`{{< image-width src="/images/blog/vs2022/vs2022-parallel-project-builds-8.png" width="100%" alt="Build and Run" >}}`